nautobot/utilities/templatetags/helpers.py

```diff
@@ -158,11 +158,7 @@ def viewname(model, action):
         >>> viewname(Device, "list")
         "dcim:device_list"
     """
-    viewname = f"{model._meta.app_label}:{model._meta.model_name}_{action}"
-    if model._meta.app_label in settings.PLUGINS:
-        viewname = f"plugins:{viewname}"
-
-    return viewname
+    return get_route_for_model(model, action)
```

Comment:
Did this fix a bug or is this cleanup?

Reply:
Both. This was the ONLY code path to get a view route that supported plugins as I had a need for this in the `DynamicGroup.map.base_url` functionality. I didn't like how it was baked into `templatetags.helpers`, so I moved it to a util function and changed the helpers to call that instead.